### PR TITLE
navbar implemented for all pages

### PR DESCRIPTION
**Pull request by Jwilson97**
created navbar and also made new file for easy render

### CODE REVIEW

**Comment by savvaq-zz** [comment]
All good!

**Comment by xinglin2016** [comment]
Seems great

**Comment by HazyHazzy** [comment]
Looks awful, did you just pull it from the wagon library and make no changes to test pull requests or something?

**Comment by Jwilson97** [comment]
> Looks awful, did you just pull it from the wagon library and make no changes to test pull requests or something?

maybe ;)

File: app/views/shared/_navbar.html.erb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<div class="navbar navbar-expand-sm navbar-light navbar-lewagon">
+  <%= link_to root_path, class: "navbar-brand" do %>
```
Comment (xinglin2016) [inline app/views/shared/_navbar.html.erb:2]: Thanks for changing the path

**Review by xinglin2016 — APPROVED** [review]

**Pull request merged**